frameworks/elastic/universe/config.json

```diff
@@ -520,7 +520,7 @@
           "default": 10000
         },
         "http_content_type_required": {
-          "description": "Enables or disables strict checking and usage of the Content-Type header for all requests with content, defaults to false.",
+          "description": "THIS FIELD IS DEPRECATED AND IS NOT BEING USED IN ANY OF THE TEMPLATE FILES. Enables or disables strict checking and usage of the Content-Type header for all requests with content, defaults to false.",
```

Comment:
@benclarkwood are you happy with how we marked this? The problem is that we "break" upgrades if we remove this field (due to `additionalProperties` being off). Do we have a ticket to track versioned schemas? Is it worth it to create one?

Reply:
This is because the field has no effect in 6.1, right @mpereira?

Reply:
@benclarkwood correct. Do you think the deprecation notice in the field description is warranted?

Reply:
I'd update it to read: "THIS FIELD IS DEPRECATED AS THIS SETTING IS NOT USED IN ELASTIC 6.1+"

Reply:
https://github.com/mesosphere/dcos-commons/pull/2078/commits/7badcc874b5084cca93b92d70556638a5aa72aac